woof-code/rootfs-petbuilds/geany-gtk2/petbuild

```diff
@@ -1,13 +1,13 @@
 download() {
-    [ -f geany-1.37.1.tar.bz2 ] || wget -t 3 -T 60 https://download.geany.org/geany-1.37.1.tar.bz2
+    [ -f geany-2.0.tar.bz2 ] || wget -t 3 -T 60 https://download.geany.org/geany-2.0.tar.bz2
 }
 
 build() {
-    tar -xjf geany-1.37.1.tar.bz2
-    cd geany-1.37.1
+    tar -xjf geany-2.0.tar.bz2
+    cd geany-2.0
     patch -p1 < ../defaults.patch
-    ./configure --prefix=/usr --disable-plugins --disable-vte --enable-gtk2
```

Comment:
Is this really required?

Reply:
https://github.com/puppylinux-woof-CE/woof-CE/pull/4218#discussion_r1466462876